_drafts/coop-values.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Coop Community Values"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags:
+  - community values
+---
+
+Having gone through a few [transitions]() in recent months along with global and local conversations surrounding the COVID-19 pandemic and rampant anti-blackness in the United States have caused those of us at the Coop to take a step back and really consider why we do what we do. From these discussions we have seen a need for and are beginning to develop the Coop's Community Values. 
+
+For full transparency and to ensure we capture the actual values of our community members we will be holding a community conversation in the next week where community members can bring their thoughts and suggestions to the table. Following this we will formalize our values statements and release a document for public comment.
```

Comment:
Given other conversations I'm having with community communication, I think we may want to:
- run the draft by a few folks privately (SciComp, Monica, etc)
- release the draft for public comment
- hold a community meeting 
- revise and release final draft

This will take out a step of release/revision. I think for the blog post what you say here is good, only reverse "release for public comment" and "hold community meeting"